tests/10-standup

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python3
+
+import amulet
+import unittest
+
+
+# Don't forget to expose using d.expose(service)
+class TestDeployment(unittest.TestCase):
+    ''' Logstash does nothing useful without being related to other
+    services. (as in, it wont even start without an input / output configured).
+    This simply validates the charm deploys. For more comprehensive testing
+    see the elk-stack bundle. https://jujucharms.com/u/containers/elk-stack
+    '''
+
+    def test_deploy(self):
+        self.deployment = amulet.Deployment(series='trusty')
+        self.deployment.add('logstash')
+        self.deployment.add('zulu8', 'cs:trusty/zulu8-2')
+        self.deployment.relate('zulu8:java', 'logstash:java')
+        self.deployment.expose('logstash')
+
+        try:
+            self.deployment.setup(timeout=900)
+            self.deployment.sentry.wait()
+        except amulet.helpers.TimeoutError:
+            msg = "Environment wasn't stood up in time"
+            amulet.raise_status(amulet.SKIP, msg=msg)
+        except:
+            raise
+
```

Comment:
These tests are a little light, I would like to see a sanity check to see if the logstash files exist in their directories, even if it does not run when related to a java system. 

However if there are comprehensive bundle tests, you can consider this a suggestion rather than a requirement.